docs/user/index.rst

```diff
@@ -108,11 +108,14 @@ to help you create fantastic documentation for your project.
    :caption: Reference
    :glob:
 
+   /reference/features
+   /glossary
```

Comment:
This seems like high billing for a glossary? I'd put it at the bottom.

Reply:
Almost at the bottom now :+1: 

I think we're going to do more refactoring of this TOC, perhaps add another sub-level.